### PR TITLE
Raise an HTTPBadRequest when trying to add a group in a group or system.Everyone to a group.

### LINKED ISSUE #794: Return 400 if a group contains system.Everyone or a group URL
Unless I'm mistaken:

* We don't support groups for anonymous requests
* We don't support recursivity in groups definitions

So we should reject with `400` if such groups definitons are created

### PR DESCRIPTION
**Pull request by Natim**
Fixes #794 

- [x] Add tests.
- [x] Add a changelog entry.

r? @leplatrem

### CODE REVIEW

File: kinto/views/groups.py [hunk 1]
```diff
@@ -21,6 +23,15 @@ def get_parent_id(self, request):
         parent_id = utils.instance_uri(request, 'bucket', id=bucket_id)
         return parent_id
 
+    def process_record(self, *args, **kwargs):
+        record = super(Group, self).process_record(*args, **kwargs)
+        for member in record['members']:
+            if member.startswith('/buckets/') or member == 'system.Everyone':
+                raise http_error(HTTPBadRequest(),
+                                 errno=ERRORS.INVALID_PARAMETERS,
+                                 message="'%s' is not a valid user ID." % member)
```
Comment (leplatrem) [inline kinto/views/groups.py]: I think we should define a colander validator on members instead.

Comment (Natim) [inline kinto/views/groups.py]: You mean using the GroupSchema to do that?

Comment (Natim) [inline kinto/views/groups.py]: Updated :D

**Comment by leplatrem** [comment]
r+ !

**Pull request merged**